Doc/library/ssl.rst

```diff
@@ -1729,6 +1779,26 @@ to speed up repeated connections from the same clients.
 
      This features requires OpenSSL 0.9.8f or newer.
 
+.. attribute:: SSLContext.highest_version
```

Comment:
I think this should be `maximum_version` for consistency with the `TLSVersion`

Reply:
The names are based on PEP 543. Also ``maximum_version`` and ``minimum_version`` are visually similar compared to ``highest_version`` and ``lowest_version``.

https://www.python.org/dev/peps/pep-0543/#configuration

```
    :param lowest_supported_version TLSVersion:
        The minimum version of TLS that should be allowed on TLS
        connections using this configuration.

    :param highest_supported_version TLSVersion:
        The maximum version of TLS that should be allowed on TLS
        connections using this configuration.
```

https://www.python.org/dev/peps/pep-0543/#tls-versions

```
class TLSVersion(Enum):
    MINIMUM_SUPPORTED = auto()
    SSLv2 = auto()
    SSLv3 = auto()
    TLSv1 = auto()
    TLSv1_1 = auto()
    TLSv1_2 = auto()
    TLSv1_3 = auto()
    MAXIMUM_SUPPORTED = auto()
```